command: nohup python tcn.py  --len_global 7 --len_local 5 --neighbor_size 1 --res 12 --cnn 6 --tcn 1 --nb_stacks 1 --lr 1e-4 --BN 1 > log/tcn 2>&1 &
time: 2019/04/21 22:06:36
--------------------------------------------------
    mask=masks[i])
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/Keras-2.2.4-py3.6.egg/keras/engine/training_utils.py", line 428, in weighted
    score_array = fn(y_true, y_pred)
  File "/home/ryj/renyajie/exp/GLST_Net/template.py", line 32, in mape
    mean_value = np.mean(y_true)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/fromnumeric.py", line 2909, in mean
    out=out, **kwargs)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/_methods.py", line 80, in _mean
    ret = ret.dtype.type(ret / rcount)
AttributeError: 'DType' object has no attribute 'type'
--------------------------------------------------
command: nohup python tcn.py  --len_global 7 --len_local 5 --neighbor_size 1 --res 12 --cnn 6 --tcn 1 --nb_stacks 2 --lr 1e-4 --BN 1 > log/tcn 2>&1 &
time: 2019/04/21 22:07:54
--------------------------------------------------
    mask=masks[i])
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/Keras-2.2.4-py3.6.egg/keras/engine/training_utils.py", line 428, in weighted
    score_array = fn(y_true, y_pred)
  File "/home/ryj/renyajie/exp/GLST_Net/template.py", line 32, in mape
    mean_value = np.mean(y_true)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/fromnumeric.py", line 2909, in mean
    out=out, **kwargs)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/_methods.py", line 80, in _mean
    ret = ret.dtype.type(ret / rcount)
AttributeError: 'DType' object has no attribute 'type'
--------------------------------------------------
command: nohup python tcn.py  --len_global 7 --len_local 5 --neighbor_size 1 --res 12 --cnn 6 --tcn 2 --nb_stacks 1 --lr 1e-4 --BN 1 > log/tcn 2>&1 &
time: 2019/04/21 22:09:15
--------------------------------------------------
    mask=masks[i])
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/Keras-2.2.4-py3.6.egg/keras/engine/training_utils.py", line 428, in weighted
    score_array = fn(y_true, y_pred)
  File "/home/ryj/renyajie/exp/GLST_Net/template.py", line 32, in mape
    mean_value = np.mean(y_true)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/fromnumeric.py", line 2909, in mean
    out=out, **kwargs)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/_methods.py", line 80, in _mean
    ret = ret.dtype.type(ret / rcount)
AttributeError: 'DType' object has no attribute 'type'
--------------------------------------------------
command: nohup python tcn.py  --len_global 7 --len_local 5 --neighbor_size 1 --res 12 --cnn 6 --tcn 2 --nb_stacks 2 --lr 1e-4 --BN 1 > log/tcn 2>&1 &
time: 2019/04/21 22:10:52
--------------------------------------------------
    mask=masks[i])
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/Keras-2.2.4-py3.6.egg/keras/engine/training_utils.py", line 428, in weighted
    score_array = fn(y_true, y_pred)
  File "/home/ryj/renyajie/exp/GLST_Net/template.py", line 32, in mape
    mean_value = np.mean(y_true)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/fromnumeric.py", line 2909, in mean
    out=out, **kwargs)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/_methods.py", line 80, in _mean
    ret = ret.dtype.type(ret / rcount)
AttributeError: 'DType' object has no attribute 'type'
--------------------------------------------------
command: nohup python tcn.py  --len_global 7 --len_local 5 --neighbor_size 2 --res 12 --cnn 6 --tcn 1 --nb_stacks 1 --lr 1e-4 --BN 1 > log/tcn 2>&1 &
time: 2019/04/21 22:12:50
--------------------------------------------------
    mask=masks[i])
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/Keras-2.2.4-py3.6.egg/keras/engine/training_utils.py", line 428, in weighted
    score_array = fn(y_true, y_pred)
  File "/home/ryj/renyajie/exp/GLST_Net/template.py", line 32, in mape
    mean_value = np.mean(y_true)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/fromnumeric.py", line 2909, in mean
    out=out, **kwargs)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/_methods.py", line 80, in _mean
    ret = ret.dtype.type(ret / rcount)
AttributeError: 'DType' object has no attribute 'type'
--------------------------------------------------
command: nohup python tcn.py  --len_global 7 --len_local 5 --neighbor_size 2 --res 12 --cnn 6 --tcn 1 --nb_stacks 2 --lr 1e-4 --BN 1 > log/tcn 2>&1 &
time: 2019/04/21 22:14:39
--------------------------------------------------
    mask=masks[i])
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/Keras-2.2.4-py3.6.egg/keras/engine/training_utils.py", line 428, in weighted
    score_array = fn(y_true, y_pred)
  File "/home/ryj/renyajie/exp/GLST_Net/template.py", line 32, in mape
    mean_value = np.mean(y_true)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/fromnumeric.py", line 2909, in mean
    out=out, **kwargs)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/_methods.py", line 80, in _mean
    ret = ret.dtype.type(ret / rcount)
AttributeError: 'DType' object has no attribute 'type'
--------------------------------------------------
command: nohup python tcn.py  --len_global 7 --len_local 5 --neighbor_size 2 --res 12 --cnn 6 --tcn 2 --nb_stacks 1 --lr 1e-4 --BN 1 > log/tcn 2>&1 &
time: 2019/04/21 22:16:31
--------------------------------------------------
    mask=masks[i])
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/Keras-2.2.4-py3.6.egg/keras/engine/training_utils.py", line 428, in weighted
    score_array = fn(y_true, y_pred)
  File "/home/ryj/renyajie/exp/GLST_Net/template.py", line 32, in mape
    mean_value = np.mean(y_true)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/fromnumeric.py", line 2909, in mean
    out=out, **kwargs)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/_methods.py", line 80, in _mean
    ret = ret.dtype.type(ret / rcount)
AttributeError: 'DType' object has no attribute 'type'
--------------------------------------------------
command: nohup python tcn.py  --len_global 7 --len_local 5 --neighbor_size 2 --res 12 --cnn 6 --tcn 2 --nb_stacks 2 --lr 1e-4 --BN 1 > log/tcn 2>&1 &
time: 2019/04/21 22:18:09
--------------------------------------------------
    mask=masks[i])
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/Keras-2.2.4-py3.6.egg/keras/engine/training_utils.py", line 428, in weighted
    score_array = fn(y_true, y_pred)
  File "/home/ryj/renyajie/exp/GLST_Net/template.py", line 32, in mape
    mean_value = np.mean(y_true)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/fromnumeric.py", line 2909, in mean
    out=out, **kwargs)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/_methods.py", line 80, in _mean
    ret = ret.dtype.type(ret / rcount)
AttributeError: 'DType' object has no attribute 'type'
--------------------------------------------------
command: nohup python tcn.py  --len_global 7 --len_local 5 --neighbor_size 3 --res 12 --cnn 6 --tcn 1 --nb_stacks 1 --lr 1e-4 --BN 1 > log/tcn 2>&1 &
time: 2019/04/21 22:19:58
--------------------------------------------------
    mask=masks[i])
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/Keras-2.2.4-py3.6.egg/keras/engine/training_utils.py", line 428, in weighted
    score_array = fn(y_true, y_pred)
  File "/home/ryj/renyajie/exp/GLST_Net/template.py", line 32, in mape
    mean_value = np.mean(y_true)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/fromnumeric.py", line 2909, in mean
    out=out, **kwargs)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/_methods.py", line 80, in _mean
    ret = ret.dtype.type(ret / rcount)
AttributeError: 'DType' object has no attribute 'type'
--------------------------------------------------
command: nohup python tcn.py  --len_global 7 --len_local 5 --neighbor_size 3 --res 12 --cnn 6 --tcn 1 --nb_stacks 2 --lr 1e-4 --BN 1 > log/tcn 2>&1 &
time: 2019/04/21 22:21:31
--------------------------------------------------
    mask=masks[i])
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/Keras-2.2.4-py3.6.egg/keras/engine/training_utils.py", line 428, in weighted
    score_array = fn(y_true, y_pred)
  File "/home/ryj/renyajie/exp/GLST_Net/template.py", line 32, in mape
    mean_value = np.mean(y_true)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/fromnumeric.py", line 2909, in mean
    out=out, **kwargs)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/_methods.py", line 80, in _mean
    ret = ret.dtype.type(ret / rcount)
AttributeError: 'DType' object has no attribute 'type'
--------------------------------------------------
command: nohup python tcn.py  --len_global 7 --len_local 5 --neighbor_size 3 --res 12 --cnn 6 --tcn 2 --nb_stacks 1 --lr 1e-4 --BN 1 > log/tcn 2>&1 &
time: 2019/04/21 22:23:11
--------------------------------------------------
    mask=masks[i])
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/Keras-2.2.4-py3.6.egg/keras/engine/training_utils.py", line 428, in weighted
    score_array = fn(y_true, y_pred)
  File "/home/ryj/renyajie/exp/GLST_Net/template.py", line 32, in mape
    mean_value = np.mean(y_true)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/fromnumeric.py", line 2909, in mean
    out=out, **kwargs)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/_methods.py", line 80, in _mean
    ret = ret.dtype.type(ret / rcount)
AttributeError: 'DType' object has no attribute 'type'
--------------------------------------------------
command: nohup python tcn.py  --len_global 7 --len_local 5 --neighbor_size 3 --res 12 --cnn 6 --tcn 2 --nb_stacks 2 --lr 1e-4 --BN 1 > log/tcn 2>&1 &
time: 2019/04/21 22:24:54
--------------------------------------------------
    mask=masks[i])
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/Keras-2.2.4-py3.6.egg/keras/engine/training_utils.py", line 428, in weighted
    score_array = fn(y_true, y_pred)
  File "/home/ryj/renyajie/exp/GLST_Net/template.py", line 32, in mape
    mean_value = np.mean(y_true)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/fromnumeric.py", line 2909, in mean
    out=out, **kwargs)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/_methods.py", line 80, in _mean
    ret = ret.dtype.type(ret / rcount)
AttributeError: 'DType' object has no attribute 'type'
--------------------------------------------------
command: nohup python tcn.py  --len_global 7 --len_local 6 --neighbor_size 1 --res 12 --cnn 6 --tcn 1 --nb_stacks 1 --lr 1e-4 --BN 1 > log/tcn 2>&1 &
time: 2019/04/21 22:26:28
--------------------------------------------------
    mask=masks[i])
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/Keras-2.2.4-py3.6.egg/keras/engine/training_utils.py", line 428, in weighted
    score_array = fn(y_true, y_pred)
  File "/home/ryj/renyajie/exp/GLST_Net/template.py", line 32, in mape
    mean_value = np.mean(y_true)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/fromnumeric.py", line 2909, in mean
    out=out, **kwargs)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/_methods.py", line 80, in _mean
    ret = ret.dtype.type(ret / rcount)
AttributeError: 'DType' object has no attribute 'type'
--------------------------------------------------
command: nohup python tcn.py  --len_global 7 --len_local 6 --neighbor_size 1 --res 12 --cnn 6 --tcn 1 --nb_stacks 2 --lr 1e-4 --BN 1 > log/tcn 2>&1 &
time: 2019/04/21 22:27:58
--------------------------------------------------
    mask=masks[i])
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/Keras-2.2.4-py3.6.egg/keras/engine/training_utils.py", line 428, in weighted
    score_array = fn(y_true, y_pred)
  File "/home/ryj/renyajie/exp/GLST_Net/template.py", line 32, in mape
    mean_value = np.mean(y_true)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/fromnumeric.py", line 2909, in mean
    out=out, **kwargs)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/_methods.py", line 80, in _mean
    ret = ret.dtype.type(ret / rcount)
AttributeError: 'DType' object has no attribute 'type'
--------------------------------------------------
command: nohup python tcn.py  --len_global 7 --len_local 6 --neighbor_size 1 --res 12 --cnn 6 --tcn 2 --nb_stacks 1 --lr 1e-4 --BN 1 > log/tcn 2>&1 &
time: 2019/04/21 22:29:29
--------------------------------------------------
    mask=masks[i])
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/Keras-2.2.4-py3.6.egg/keras/engine/training_utils.py", line 428, in weighted
    score_array = fn(y_true, y_pred)
  File "/home/ryj/renyajie/exp/GLST_Net/template.py", line 32, in mape
    mean_value = np.mean(y_true)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/fromnumeric.py", line 2909, in mean
    out=out, **kwargs)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/_methods.py", line 80, in _mean
    ret = ret.dtype.type(ret / rcount)
AttributeError: 'DType' object has no attribute 'type'
--------------------------------------------------
command: nohup python tcn.py  --len_global 7 --len_local 6 --neighbor_size 1 --res 12 --cnn 6 --tcn 2 --nb_stacks 2 --lr 1e-4 --BN 1 > log/tcn 2>&1 &
time: 2019/04/21 22:31:03
--------------------------------------------------
    mask=masks[i])
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/Keras-2.2.4-py3.6.egg/keras/engine/training_utils.py", line 428, in weighted
    score_array = fn(y_true, y_pred)
  File "/home/ryj/renyajie/exp/GLST_Net/template.py", line 32, in mape
    mean_value = np.mean(y_true)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/fromnumeric.py", line 2909, in mean
    out=out, **kwargs)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/_methods.py", line 80, in _mean
    ret = ret.dtype.type(ret / rcount)
AttributeError: 'DType' object has no attribute 'type'
--------------------------------------------------
command: nohup python tcn.py  --len_global 7 --len_local 6 --neighbor_size 2 --res 12 --cnn 6 --tcn 1 --nb_stacks 1 --lr 1e-4 --BN 1 > log/tcn 2>&1 &
time: 2019/04/21 22:32:46
--------------------------------------------------
    mask=masks[i])
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/Keras-2.2.4-py3.6.egg/keras/engine/training_utils.py", line 428, in weighted
    score_array = fn(y_true, y_pred)
  File "/home/ryj/renyajie/exp/GLST_Net/template.py", line 32, in mape
    mean_value = np.mean(y_true)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/fromnumeric.py", line 2909, in mean
    out=out, **kwargs)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/_methods.py", line 80, in _mean
    ret = ret.dtype.type(ret / rcount)
AttributeError: 'DType' object has no attribute 'type'
--------------------------------------------------
command: nohup python tcn.py  --len_global 7 --len_local 6 --neighbor_size 2 --res 12 --cnn 6 --tcn 1 --nb_stacks 2 --lr 1e-4 --BN 1 > log/tcn 2>&1 &
time: 2019/04/21 22:34:17
--------------------------------------------------
    mask=masks[i])
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/Keras-2.2.4-py3.6.egg/keras/engine/training_utils.py", line 428, in weighted
    score_array = fn(y_true, y_pred)
  File "/home/ryj/renyajie/exp/GLST_Net/template.py", line 32, in mape
    mean_value = np.mean(y_true)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/fromnumeric.py", line 2909, in mean
    out=out, **kwargs)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/_methods.py", line 80, in _mean
    ret = ret.dtype.type(ret / rcount)
AttributeError: 'DType' object has no attribute 'type'
--------------------------------------------------
command: nohup python tcn.py  --len_global 7 --len_local 6 --neighbor_size 2 --res 12 --cnn 6 --tcn 2 --nb_stacks 1 --lr 1e-4 --BN 1 > log/tcn 2>&1 &
time: 2019/04/21 22:35:48
--------------------------------------------------
    mask=masks[i])
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/Keras-2.2.4-py3.6.egg/keras/engine/training_utils.py", line 428, in weighted
    score_array = fn(y_true, y_pred)
  File "/home/ryj/renyajie/exp/GLST_Net/template.py", line 32, in mape
    mean_value = np.mean(y_true)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/fromnumeric.py", line 2909, in mean
    out=out, **kwargs)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/_methods.py", line 80, in _mean
    ret = ret.dtype.type(ret / rcount)
AttributeError: 'DType' object has no attribute 'type'
--------------------------------------------------
command: nohup python tcn.py  --len_global 7 --len_local 6 --neighbor_size 2 --res 12 --cnn 6 --tcn 2 --nb_stacks 2 --lr 1e-4 --BN 1 > log/tcn 2>&1 &
time: 2019/04/21 22:37:25
--------------------------------------------------
    mask=masks[i])
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/Keras-2.2.4-py3.6.egg/keras/engine/training_utils.py", line 428, in weighted
    score_array = fn(y_true, y_pred)
  File "/home/ryj/renyajie/exp/GLST_Net/template.py", line 32, in mape
    mean_value = np.mean(y_true)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/fromnumeric.py", line 2909, in mean
    out=out, **kwargs)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/_methods.py", line 80, in _mean
    ret = ret.dtype.type(ret / rcount)
AttributeError: 'DType' object has no attribute 'type'
--------------------------------------------------
command: nohup python tcn.py  --len_global 7 --len_local 6 --neighbor_size 3 --res 12 --cnn 6 --tcn 1 --nb_stacks 1 --lr 1e-4 --BN 1 > log/tcn 2>&1 &
time: 2019/04/21 22:39:05
--------------------------------------------------
    mask=masks[i])
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/Keras-2.2.4-py3.6.egg/keras/engine/training_utils.py", line 428, in weighted
    score_array = fn(y_true, y_pred)
  File "/home/ryj/renyajie/exp/GLST_Net/template.py", line 32, in mape
    mean_value = np.mean(y_true)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/fromnumeric.py", line 2909, in mean
    out=out, **kwargs)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/_methods.py", line 80, in _mean
    ret = ret.dtype.type(ret / rcount)
AttributeError: 'DType' object has no attribute 'type'
--------------------------------------------------
command: nohup python tcn.py  --len_global 7 --len_local 6 --neighbor_size 3 --res 12 --cnn 6 --tcn 1 --nb_stacks 2 --lr 1e-4 --BN 1 > log/tcn 2>&1 &
time: 2019/04/21 22:40:41
--------------------------------------------------
    mask=masks[i])
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/Keras-2.2.4-py3.6.egg/keras/engine/training_utils.py", line 428, in weighted
    score_array = fn(y_true, y_pred)
  File "/home/ryj/renyajie/exp/GLST_Net/template.py", line 32, in mape
    mean_value = np.mean(y_true)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/fromnumeric.py", line 2909, in mean
    out=out, **kwargs)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/_methods.py", line 80, in _mean
    ret = ret.dtype.type(ret / rcount)
AttributeError: 'DType' object has no attribute 'type'
--------------------------------------------------
command: nohup python tcn.py  --len_global 7 --len_local 6 --neighbor_size 3 --res 12 --cnn 6 --tcn 2 --nb_stacks 1 --lr 1e-4 --BN 1 > log/tcn 2>&1 &
time: 2019/04/21 22:42:19
--------------------------------------------------
    mask=masks[i])
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/Keras-2.2.4-py3.6.egg/keras/engine/training_utils.py", line 428, in weighted
    score_array = fn(y_true, y_pred)
  File "/home/ryj/renyajie/exp/GLST_Net/template.py", line 32, in mape
    mean_value = np.mean(y_true)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/fromnumeric.py", line 2909, in mean
    out=out, **kwargs)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/_methods.py", line 80, in _mean
    ret = ret.dtype.type(ret / rcount)
AttributeError: 'DType' object has no attribute 'type'
--------------------------------------------------
command: nohup python tcn.py  --len_global 7 --len_local 6 --neighbor_size 3 --res 12 --cnn 6 --tcn 2 --nb_stacks 2 --lr 1e-4 --BN 1 > log/tcn 2>&1 &
time: 2019/04/21 22:44:04
--------------------------------------------------
    mask=masks[i])
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/Keras-2.2.4-py3.6.egg/keras/engine/training_utils.py", line 428, in weighted
    score_array = fn(y_true, y_pred)
  File "/home/ryj/renyajie/exp/GLST_Net/template.py", line 32, in mape
    mean_value = np.mean(y_true)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/fromnumeric.py", line 2909, in mean
    out=out, **kwargs)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/_methods.py", line 80, in _mean
    ret = ret.dtype.type(ret / rcount)
AttributeError: 'DType' object has no attribute 'type'
--------------------------------------------------
command: nohup python tcn.py  --len_global 7 --len_local 7 --neighbor_size 1 --res 12 --cnn 6 --tcn 1 --nb_stacks 1 --lr 1e-4 --BN 1 > log/tcn 2>&1 &
time: 2019/04/21 22:45:57
--------------------------------------------------
    mask=masks[i])
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/Keras-2.2.4-py3.6.egg/keras/engine/training_utils.py", line 428, in weighted
    score_array = fn(y_true, y_pred)
  File "/home/ryj/renyajie/exp/GLST_Net/template.py", line 32, in mape
    mean_value = np.mean(y_true)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/fromnumeric.py", line 2909, in mean
    out=out, **kwargs)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/_methods.py", line 80, in _mean
    ret = ret.dtype.type(ret / rcount)
AttributeError: 'DType' object has no attribute 'type'
--------------------------------------------------
command: nohup python tcn.py  --len_global 7 --len_local 7 --neighbor_size 1 --res 12 --cnn 6 --tcn 1 --nb_stacks 2 --lr 1e-4 --BN 1 > log/tcn 2>&1 &
time: 2019/04/21 22:47:32
--------------------------------------------------
    mask=masks[i])
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/Keras-2.2.4-py3.6.egg/keras/engine/training_utils.py", line 428, in weighted
    score_array = fn(y_true, y_pred)
  File "/home/ryj/renyajie/exp/GLST_Net/template.py", line 32, in mape
    mean_value = np.mean(y_true)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/fromnumeric.py", line 2909, in mean
    out=out, **kwargs)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/_methods.py", line 80, in _mean
    ret = ret.dtype.type(ret / rcount)
AttributeError: 'DType' object has no attribute 'type'
--------------------------------------------------
command: nohup python tcn.py  --len_global 7 --len_local 7 --neighbor_size 1 --res 12 --cnn 6 --tcn 2 --nb_stacks 1 --lr 1e-4 --BN 1 > log/tcn 2>&1 &
time: 2019/04/21 22:49:16
--------------------------------------------------
    mask=masks[i])
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/Keras-2.2.4-py3.6.egg/keras/engine/training_utils.py", line 428, in weighted
    score_array = fn(y_true, y_pred)
  File "/home/ryj/renyajie/exp/GLST_Net/template.py", line 32, in mape
    mean_value = np.mean(y_true)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/fromnumeric.py", line 2909, in mean
    out=out, **kwargs)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/_methods.py", line 80, in _mean
    ret = ret.dtype.type(ret / rcount)
AttributeError: 'DType' object has no attribute 'type'
--------------------------------------------------
command: nohup python tcn.py  --len_global 7 --len_local 7 --neighbor_size 1 --res 12 --cnn 6 --tcn 2 --nb_stacks 2 --lr 1e-4 --BN 1 > log/tcn 2>&1 &
time: 2019/04/21 22:51:01
--------------------------------------------------
    mask=masks[i])
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/Keras-2.2.4-py3.6.egg/keras/engine/training_utils.py", line 428, in weighted
    score_array = fn(y_true, y_pred)
  File "/home/ryj/renyajie/exp/GLST_Net/template.py", line 32, in mape
    mean_value = np.mean(y_true)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/fromnumeric.py", line 2909, in mean
    out=out, **kwargs)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/_methods.py", line 80, in _mean
    ret = ret.dtype.type(ret / rcount)
AttributeError: 'DType' object has no attribute 'type'
--------------------------------------------------
command: nohup python tcn.py  --len_global 7 --len_local 7 --neighbor_size 2 --res 12 --cnn 6 --tcn 1 --nb_stacks 1 --lr 1e-4 --BN 1 > log/tcn 2>&1 &
time: 2019/04/21 22:52:48
--------------------------------------------------
    mask=masks[i])
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/Keras-2.2.4-py3.6.egg/keras/engine/training_utils.py", line 428, in weighted
    score_array = fn(y_true, y_pred)
  File "/home/ryj/renyajie/exp/GLST_Net/template.py", line 32, in mape
    mean_value = np.mean(y_true)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/fromnumeric.py", line 2909, in mean
    out=out, **kwargs)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/_methods.py", line 80, in _mean
    ret = ret.dtype.type(ret / rcount)
AttributeError: 'DType' object has no attribute 'type'
--------------------------------------------------
command: nohup python tcn.py  --len_global 7 --len_local 7 --neighbor_size 2 --res 12 --cnn 6 --tcn 1 --nb_stacks 2 --lr 1e-4 --BN 1 > log/tcn 2>&1 &
time: 2019/04/21 22:54:20
--------------------------------------------------
    mask=masks[i])
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/Keras-2.2.4-py3.6.egg/keras/engine/training_utils.py", line 428, in weighted
    score_array = fn(y_true, y_pred)
  File "/home/ryj/renyajie/exp/GLST_Net/template.py", line 32, in mape
    mean_value = np.mean(y_true)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/fromnumeric.py", line 2909, in mean
    out=out, **kwargs)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/_methods.py", line 80, in _mean
    ret = ret.dtype.type(ret / rcount)
AttributeError: 'DType' object has no attribute 'type'
--------------------------------------------------
command: nohup python tcn.py  --len_global 7 --len_local 7 --neighbor_size 2 --res 12 --cnn 6 --tcn 2 --nb_stacks 1 --lr 1e-4 --BN 1 > log/tcn 2>&1 &
time: 2019/04/21 22:55:54
--------------------------------------------------
    mask=masks[i])
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/Keras-2.2.4-py3.6.egg/keras/engine/training_utils.py", line 428, in weighted
    score_array = fn(y_true, y_pred)
  File "/home/ryj/renyajie/exp/GLST_Net/template.py", line 32, in mape
    mean_value = np.mean(y_true)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/fromnumeric.py", line 2909, in mean
    out=out, **kwargs)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/_methods.py", line 80, in _mean
    ret = ret.dtype.type(ret / rcount)
AttributeError: 'DType' object has no attribute 'type'
--------------------------------------------------
command: nohup python tcn.py  --len_global 7 --len_local 7 --neighbor_size 2 --res 12 --cnn 6 --tcn 2 --nb_stacks 2 --lr 1e-4 --BN 1 > log/tcn 2>&1 &
time: 2019/04/21 22:57:36
--------------------------------------------------
    mask=masks[i])
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/Keras-2.2.4-py3.6.egg/keras/engine/training_utils.py", line 428, in weighted
    score_array = fn(y_true, y_pred)
  File "/home/ryj/renyajie/exp/GLST_Net/template.py", line 32, in mape
    mean_value = np.mean(y_true)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/fromnumeric.py", line 2909, in mean
    out=out, **kwargs)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/_methods.py", line 80, in _mean
    ret = ret.dtype.type(ret / rcount)
AttributeError: 'DType' object has no attribute 'type'
--------------------------------------------------
command: nohup python tcn.py  --len_global 7 --len_local 7 --neighbor_size 3 --res 12 --cnn 6 --tcn 1 --nb_stacks 1 --lr 1e-4 --BN 1 > log/tcn 2>&1 &
time: 2019/04/21 22:59:19
--------------------------------------------------
    mask=masks[i])
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/Keras-2.2.4-py3.6.egg/keras/engine/training_utils.py", line 428, in weighted
    score_array = fn(y_true, y_pred)
  File "/home/ryj/renyajie/exp/GLST_Net/template.py", line 32, in mape
    mean_value = np.mean(y_true)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/fromnumeric.py", line 2909, in mean
    out=out, **kwargs)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/_methods.py", line 80, in _mean
    ret = ret.dtype.type(ret / rcount)
AttributeError: 'DType' object has no attribute 'type'
--------------------------------------------------
command: nohup python tcn.py  --len_global 7 --len_local 7 --neighbor_size 3 --res 12 --cnn 6 --tcn 1 --nb_stacks 2 --lr 1e-4 --BN 1 > log/tcn 2>&1 &
time: 2019/04/21 23:01:02
--------------------------------------------------
    mask=masks[i])
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/Keras-2.2.4-py3.6.egg/keras/engine/training_utils.py", line 428, in weighted
    score_array = fn(y_true, y_pred)
  File "/home/ryj/renyajie/exp/GLST_Net/template.py", line 32, in mape
    mean_value = np.mean(y_true)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/fromnumeric.py", line 2909, in mean
    out=out, **kwargs)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/_methods.py", line 80, in _mean
    ret = ret.dtype.type(ret / rcount)
AttributeError: 'DType' object has no attribute 'type'
--------------------------------------------------
command: nohup python tcn.py  --len_global 7 --len_local 7 --neighbor_size 3 --res 12 --cnn 6 --tcn 2 --nb_stacks 1 --lr 1e-4 --BN 1 > log/tcn 2>&1 &
time: 2019/04/21 23:02:46
--------------------------------------------------
    mask=masks[i])
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/Keras-2.2.4-py3.6.egg/keras/engine/training_utils.py", line 428, in weighted
    score_array = fn(y_true, y_pred)
  File "/home/ryj/renyajie/exp/GLST_Net/template.py", line 32, in mape
    mean_value = np.mean(y_true)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/fromnumeric.py", line 2909, in mean
    out=out, **kwargs)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/_methods.py", line 80, in _mean
    ret = ret.dtype.type(ret / rcount)
AttributeError: 'DType' object has no attribute 'type'
--------------------------------------------------
command: nohup python tcn.py  --len_global 7 --len_local 7 --neighbor_size 3 --res 12 --cnn 6 --tcn 2 --nb_stacks 2 --lr 1e-4 --BN 1 > log/tcn 2>&1 &
time: 2019/04/21 23:04:30
--------------------------------------------------
    mask=masks[i])
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/Keras-2.2.4-py3.6.egg/keras/engine/training_utils.py", line 428, in weighted
    score_array = fn(y_true, y_pred)
  File "/home/ryj/renyajie/exp/GLST_Net/template.py", line 32, in mape
    mean_value = np.mean(y_true)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/fromnumeric.py", line 2909, in mean
    out=out, **kwargs)
  File "/home/ryj/anaconda3/envs/ryj_keras/lib/python3.6/site-packages/numpy/core/_methods.py", line 80, in _mean
    ret = ret.dtype.type(ret / rcount)
AttributeError: 'DType' object has no attribute 'type'
--------------------------------------------------
